src/components/Addbook.js

```diff
@@ -1,39 +1,72 @@
-import { React, useState } from 'react';
+import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
 import { addbook } from '../redux/books/books';
 
 function Addbook() {
   const dispatch = useDispatch();
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
+  const [bookData, setBookData] = useState({
+    id: uuidv4(),
+    title: '',
+    author: '',
+    category: '',
+  });
 
   const formHandler = (e) => {
     e.preventDefault();
-    dispatch(
-      addbook({
-        id: uuidv4(),
-        title,
-        author,
-      }),
-    );
-    setTitle('');
-    setAuthor('');
+    dispatch(addbook(bookData));
+    setBookData({
+      id: uuidv4(),
+      title: '',
+      author: '',
+      category: '',
+    });
+    // dispatch(
+    //   addbook({
+    //     id: uuidv4(),
+    //     title,
+    //     author,
+    //   }),
+    // );
+    // setTitle('');
+    // setAuthor('');
   };
 
-  const titleHandler = (e) => {
-    setTitle(e.target.value);
+  // const titleHandler = (e) => {
+  // setTitle(e.target.value);
```

Comment:
- [ ] Kindly make sure you remove all the commented code lines from your codebase to make it cleaner, more professional and for **[best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**